### PR TITLE
Zcbor 0.5.1

### PR DESCRIPTION
**Pull request by oyvindronningstad**
Update zcbor from 0.4.0 to 0.5.1.

### CODE REVIEW

**Review by FrancescoSer — APPROVED** [review]

**Comment by oyvindronningstad** [comment]
@shantha-14 Who should review the requirements*.txt changes?

**Comment by oyvindronningstad** [comment]
Squashed into 1 commit.

**Review by carlescufi — APPROVED** [review]

**Comment by oyvindronningstad** [comment]
@shantha-14 I have run CI-all-test on this, with two failures, both of which have been confirmed as unrelated (cc @richardmccrae-nordicsemi @magnev).

Can you or someone else review the requirements*.txt changes?

File: tests/modules/lib/zcbor/cddl/test.cddl [hunk 1]
```diff
@@ -8,6 +8,6 @@
 Timestamp = bstr .size 8
 Test = [
     name: [ +tstr ],
-    time: Timestamp,
+    timestamp: Timestamp,
```
Comment (hakonfam) [inline tests/modules/lib/zcbor/cddl/test.cddl:11]: Just curious, why did we rename this?

**Review by hakonfam — APPROVED** [review]
fmfu_fdev and ztest looks good.

**Review by shantha-14 — APPROVED** [review]
@carlescufi  or @mbolivar-nordic  this PR can be merged, the RED CI is actually green on re-run

**Pull request merged**